core/domain/stats_domain.py

```diff
@@ -1354,9 +1495,11 @@ def to_raw_type(self):
         }
 
     @classmethod
-    def from_raw_type(cls, answer_occurrence_dict):
-        """Returns domain object that represents a specific answer that occurred
-        some number of times.
+    def from_raw_type(
+        cls, answer_occurrence_dict:
```

Comment:
Possible to move answer_occurrence_dict to next line?

Reply:
Done